initrd/bin/tpmr

```diff
@@ -533,6 +533,7 @@ tpm2_unseal() {
 	# can't do anything without a primary handle.
 	if [ ! -f "/tmp/$PRIMARY_HANDLE_FILE" ]; then
 		DEBUG "tpm2_unseal: No primary handle, cannot attempt to unseal"
+		warn "No TPM primary handle. You must reset TPM to seal secret"
```

Comment:
@JonathonHall-Purism this helps the user IMO.

Reply:
Agree, much better than just a cryptic error.